3422.xml

```diff
@@ -0,0 +1,398 @@
+		<ObjectID>3422</ObjectID>
+		<ObjectURN>urn:oma:lwm2m:ext:3422</ObjectURN>
+		<LWM2MVersion>1.0</LWM2MVersion>
+		<ObjectVersion>1.0</ObjectVersion>
+		<MultipleInstances>Multiple</MultipleInstances>
+		<Mandatory>Optional</Mandatory>
+		<Resources>
+			<Item ID="1">
+				<Name>L1-L2 Voltage</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>V</Units>
+				<Description>Voltage measured between line 1 and line 2.</Description>
+			</Item>
+			<Item ID="2">
+				<Name>L2-L3 Voltage</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>V</Units>
+				<Description>Voltage measured between line 2 and line 3.</Description>
+			</Item>
+			<Item ID="3">
+				<Name>L3-L1 Voltage</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>V</Units>
+				<Description>Voltage measured between line 3 and line 1.</Description>
+			</Item>
+			<Item ID="4">
+				<Name>Total Current of all three phases</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>A</Units>
+				<Description>Sum of current measured on all three lines.</Description>
+			</Item>
+			<Item ID="5">
+				<Name>Instantaneous Power Factor overall</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Instantaneous power factor overall.</Description>
+			</Item>
+			<Item ID="6">
+				<Name>Last Average Power Factor</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Last average power factor calculated from each power factor on the 3 lines.</Description>
+			</Item>
+			<Item ID="7">
+				<Name>Minimum Power Factor</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Minimum power factor measured on the 3 lines.</Description>
+			</Item>
+			<Item ID="8">
+				<Name>System kW: Inst. Value. ΣLi Active power+ (QI+QIV)</Name>
```

Comment:
The `Σ` character is likely to cause (even more than the parenthesis and +/-) problems for some uses. In the description it's less problematic, but there you even better is to write it out. If there's OK way to write the names in this object without that character, I would suggest changing that.